engine/src/canvas.mlc

```diff
@@ -3729,6 +3729,10 @@ Description:
 
 >*Note:* The layout bounds is used to position blocks of text relative to each other, so the text may overlap these bounds.
 
+The `layout` is optional and has no effect on the return value, but may be used to emphasize the difference between `the layout bounds` and `the image bounds`.
```

Comment:
I'm not sure this is the right way to explain this. The syntax is 'the [layout] bounds' which indicates that the layout token is optional in this context - this is a standard formalism from LCS and if we were to mention this kind of thing in every single place where there is an optional token there would be way too much verbiage in docs which add no value.